source/client/controllers/login-controller.js

```diff
@@ -8,7 +8,8 @@ Space.messaging.Controller.extend(Space.accountsUi, 'LoginController', {
   eventSubscriptions() {
     return [{
       'Space.accountsUi.LoginRequested': this._onLoginRequested,
-      'Space.accountsUi.LogoutRequested': this._onLogoutRequested
+      'Space.accountsUi.LogoutRequested': this._onLogoutRequested,
+      'Space.accountsUi.LoginWithGoogleRequested': this._onLoginWithGoogleRequested,
```

Comment:
I would rather keep a single `LoginRequested` event and use something like a `service` property to distinguish between the various strategies. Same for the failed / succeeded events!